cpp/tests/test_odesecir_ageres.cpp

```diff
@@ -24,8 +24,12 @@
 #include "memilio/math/stepper_wrapper.h"
 #include <gtest/gtest.h>
 
-TEST(TestSecir, compareAgeResWithSingleRun)
+TEST(TestOdeSecir, compareAgeResWithPreviousRun)
 {
+    /*
```

Comment:
I think the idea is good, but i doubt that someone will compare the results from both files. Is there any way to do this as part of the ci?

Reply:
I dont think anyone would need to compare the results from both files. Thats done by the tests. the comment is only a hint towards the fact that there exists a similar test, that compares the c++ simulation results with the python simulation and that the python test is likely to fail, if the c++ test is failing.

Reply:
I dont know if and how something like this could be implemented in the ci.